content/faq/how-do-I-use-composer-with-github-access-token.md

```diff
@@ -0,0 +1,51 @@
+---
+title: How do I use Composer with GitHub personal access token?
+menu:
+  main:
+    parent: 'faq'
+    weight: 40
+weight: 40
+linkTitle: Setup GitHub Token
+---
+If you need to download private repositories from GitHub during the build
+process, an access token has to be available during `composer install`.
+
+To set up that, make sure you are using one of the Docker images from
+[ContinuousPipe dockerfiles](https://github.com/continuouspipe/dockerfiles)
+as base image. These images are prepared to take a GITHUB_TOKEN 
+build argument which is used to set the token like this:
+`composer global config github-oauth.github.com "$GITHUB_TOKEN"`
+
+{{< warning title="Warning" >}}
+Token can be added as either a Docker build argument or an environment 
+variable. When it is passed as a build argument, the token will be visible
+in the image history (see `docker history <image-name>` command). When it 
+is given as an environment variable, the token is added to the image 
+itself (see `docker inspect --format '{{ index (index .Config.Env) }}' <image-name>`
+).
+However the composer call above will save it in the filesystem of the image.
```

Comment:
Perhaps rephrase this paragraph to say "With all of these methods of exposing the token, the final Docker image should be pushed into a private repository to avoid leaking the Github token."